native/src/main/java/org/ballerinax/asb/receiver/MessageReceiver.java

```diff
@@ -603,6 +617,13 @@ private static void populateApplicationProperty(BMap<BString, Object> applicatio
         }
     }
 
+    private static ServiceBusReceiverClient getReceiverFromBObject(BObject bReceiver, boolean isDeadLetter) {
+        if (isDeadLetter) {
+            return (ServiceBusReceiverClient) getDeadLetterMessageReceiverFromBObject(bReceiver);
+        }
+        return (ServiceBusReceiverClient) bReceiver.getNativeData(RECEIVER_CLIENT);
```

Comment:
```suggestion
        return getReceiverFromBObject(bReceiver);
```

Reply:
Rather than `getReceiverFromBObject` I think more intuitive method name would be `getNativeReceiver`. And since there is no additional benefit of calling another method just to get the native client in non-dead letter scenario, thought of refactor the current naming and use it in everyplace. 

check this commit: [f0e504c63ed0871f5de284d12e3d32680f3d135e](https://github.com/ballerina-platform/module-ballerinax-azure-service-bus/pull/230/commits/f0e504c63ed0871f5de284d12e3d32680f3d135e)

Reply:
+1